app/run_rake_task.rb

```diff
@@ -23,6 +23,27 @@ def self.links(application, rake_task = "")
     ERB
   end
 
+  def self.terse_links(application, rake_task = "")
+    application = find_application(application) if application.is_a? String
+
+    rake_task_name = rake_task.presence || "a Rake task"
+
+    integration_link = "<a
```

Comment:
Are we able to use the [`link_to`](https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to) helper here?  It might mean we can remove the need to escape quotes.

Reply:
I think I ran into two problems with that:
1. Trying to call it from outside the partial
2. It not supporting the `aria-label` property.